### ISSUE TITLE
No auto-imports for amethyst crate

### ISSUE THREAD

**Issue by TransmissionsDev** [intellij-rust/intellij-rust#5749]
## Environment

* **IntelliJ Rust plugin version:**  0.3.126.3220-201
* **Rust toolchain version:** 1.44.1
* **IDE name and version:** CLION 2020.1
* **Operating system:** MacOS 10.15.4

## Problem description

Any code from the `amethyst` game engine crate is not giving me auto imports like every other crate does.

- I normally see error messages like this on hovering over an unimported function or type:

![ScreenShot-](https://user-images.githubusercontent.com/26209401/87713501-297b7a00-c75f-11ea-998b-f9d71cca169a.png)

- But with all exports from the `amethyst` create I only see this:

![ScreenShot-](https://user-images.githubusercontent.com/26209401/87713544-457f1b80-c75f-11ea-81cf-656d1ed83525.png)

## Steps to reproduce

```rust
use amethyst::input::StringBindings;

fn main() {
    let input_bundle = InputBundle::<StringBindings>::new();
}
```

If you put this code into your `main.rs` file in a project with the `amethyst` crate in your `Cargo.toml` and hover over InputBundle you will see: 

![ScreenShot-](https://user-images.githubusercontent.com/26209401/87713544-457f1b80-c75f-11ea-81cf-656d1ed83525.png)

**You can also try importing just the InputBundle but not StringBindings and will also not get auto-imports.**

## More Info

What's even more confusing is I do get definitions for imported functions just no auto import suggestions before importing:
![ScreenShot-](https://user-images.githubusercontent.com/26209401/87713754-92fb8880-c75f-11ea-8112-304ac61bf2fd.png)

**Comment by pikapak** [comment]
Same issue here.

Environment:
- IntelliJ Rust plugin version: 0.3.126.3220-201
- Rust toolchain version: 1.45.0
- IDE name and version: CLION 2020.1.2
- Operating system: Windows 10

**Comment by TransmissionsDev** [comment]
Is there any workarounds for this? It's really confusing to figure out how to import from Amethyst and I really could use the help of auto import.

**Comment by onelson** [comment]
I've just been looking up the module scopes for the symbols I need in the API docs as I go. It's manual, but it works.

**Comment by TransmissionsDev** [comment]
> I've just been looking up the module scopes for the symbols I need in the API docs as I go. It's manual, but it works.

I've been trying to do this but all the symbols are scattered around different crates and re-exported, so how do you know how to get the actual importable path?

**Comment by onelson** [comment]
I think the only rule of thumb I keep in my mind is: crates that are named `amethyst_foo` are re-exported as `amethyst::foo`. From there, the module path should be the same.

The only other issue I've had is trying to access modules or symbols that are only available when certain features are enabled. For that, I inspect the amethyst `Cargo.toml` for the feature list to see if there's a clue. For instance, the `amethyst_gltf` crate might only be re-exported when the `gltf` feature is enabled.

**Comment by Undin** [comment]
Thanks for your feedback!
The issue should be fixed in #5757

**Comment by TransmissionsDev** [comment]
Wohoo! That was fast! Thank you @Undin !
